pages/user-interfaces.html

```diff
@@ -0,0 +1,163 @@
+<div class="frameworkcontent">
+
+<p>
+If you don't want to use any of the previously mentioned editors, you
+can
+use <a href="https://coq.inria.fr/refman/practical-tools/coqide.html">CoqIDE</a>,
+which is developed and distributed alongside Coq.  Like most of the
+more modern editor extensions, CoqIDE relies
+on <a href="https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md">Coq's
+XML protocol for IDEs</a>, and implements all of the features provided
+by this protocol.  On the other hand, it does not support automatic
+indentation contrarily to most editor extensions.
+</p>
+
+<p>
+As a way to try Coq without installing anything, you can
+use <a href="https://jscoq.github.io/">JsCoq</a>.  JsCoq loads Coq
+entirely in your browser.  However, it is too limited to conduct
+actual projects: it lacks access to the VM and native computing
+machineries of Coq, and may hit out-of-memory and stack-overflow
+failures quicker than native versions of Coq.
+</p>
+
+</div>
+
+<div class="frameworklinks">
+<ul>
+<li><a href="refman/practical-tools/coqide.html">CoqIDE</a></li>
+<li><a class="extlink" href="https://jscoq.github.io/">JsCoq</a></li>
+</ul>
+</div>
+</div>
+
+<div class="framework">
+<div class="frameworklabel">Experimental / discontinued interfaces</div>
+<div class="frameworkcontent">
+
+<p>
+There have been many experiments over the years.  Here are some
+additional user interfaces that have stayed at the experimental level
+or whose maintenance has been discontinued and or is currently
+uncertain:
+<ul>
+<li>
+Proof General also has an
+experimental <a href="https://github.com/ProofGeneral/PG/tree/async">async
+branch</a> that relies
+on <a href="https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md">Coq's
+XML protocol for IDEs</a>, and supports new features such as
+asynchronous proof processing.  Unfortunately, this branch is mostly
+unmaintained and quite unstable.
+</li>
+<li>
+The <a href="http://coqoon.github.io/">Coqoon</a> Eclipse plugin was
+initially based upon
+the <a href="https://bitbucket.org/coqpide/pidetop/src/PIDEtop/">PIDEtop
+Coq plugin</a> (following Wenzel's asynchronous PIDE framework).
+Support for more recent version of Coq (8.7 and 8.8) relies
+on <a href="https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md">Coq's
+XML protocol for IDEs</a>.
+</li>
+<li>
+The <a href="https://github.com/Ptival/PeaCoq">PeaCoq</a> online web
+interface for Coq was focused on teaching (actively developed from
+2014 to 2016).
+</li>
+<li>
+The <a href="http://prover.cs.ru.nl">ProofWeb</a> online web interface
+for Coq (and other proof assistants) was also focused on teaching (in
+2006-2007).
+</li>
+<li>
+<a href="http://provereditor.gforge.inria.fr">ProverEditor</a> was an
+experimental Eclipse plugin with support for Coq (in 2005-2006).
+</li>
+<li>
+<a href="https://www-sop.inria.fr/lemme/pcoq/">Pcoq</a> (discontinued
+in 2003) was a first experiment at proof-by-pointing.
```

Comment:
Add a link to explain proof-by-pointing? https://www-sop.inria.fr/croap/ctcoq/help/pbp.html ?